src/Microsoft.Identity.Abstractions/DownstreamRestApi/DownstreamRestApiOptions.cs

```diff
@@ -26,7 +54,18 @@ public class DownstreamRestApiOptions
         /// <summary>
         /// HTTP method used to call this downstream web API (by default Get).
         /// </summary>
-        public HttpMethod HttpMethod { get; set; } = HttpMethod.Get;
+        [DefaultValue("Get")]
```

Comment:
What is it for?

Reply:
This provides information to developers.